components/Starknet/modules/architecture-and-concepts/pages/network-architecture/messaging-mechanism.adoc

```diff
@@ -221,8 +222,9 @@ l1_handler_tx_hash = ℎ(
 Where:
 
 - `l1_handler` is a constant prefix, encoded in bytes (ASCII), as big-endian.
+- `version` is the transaction version. Only version 0 is currently supported.
```

Comment:
We may refer to the new table to have this being true forever, even if a new version shows up?